pkg/proxy/ipvs/proxier.go

```diff
@@ -153,17 +153,17 @@ func (r *realIPGetter) NodeIPs() (ips []net.IP, err error) {
 		}
 		intf, err := net.InterfaceByName(name)
 		if err != nil {
+			glog.Errorf("Failed to get interface by name: %s, error: %v", name, err)
```

Comment:
Always use utilruntime.HandleError if you are eating an error.

Reply:
It's fixed now. PTAL. Thanks!